tests/VirtoCommerce.Platform.Core.Tests/Common/ChangeDetectorTests.cs

```diff
@@ -0,0 +1,51 @@
+using VirtoCommerce.Platform.Core.ChangesUtils;
+using Xunit;
+using FluentAssertions;
+
+namespace VirtoCommerce.Platform.Core.Tests.Common
+{
+    public class ChangeDetectorTests
```

Comment:
We need to add a test case:
1. Parent class has predefined properties with marked attributes (for ex. Prop1, Prop2);
2. Child class has own properties (Prop3, Prop4);
3. We have different values for Prop1, Prop2, Prop3, Prop4;
4. Collect changes for **Child** class;
5. Ensure we collect changes only for Prop3 and Prop4.